source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -188,17 +188,17 @@ void BioXASAppController::onRegionOfInterestBoundingRangeChanged(AMRegionOfInter
 		xasConfiguration_->setRegionOfInterestBoundingRange(region);
 }
 
-void BioXASAppController::goToBeamStatusView(AMControl *control)
+void BioXASAppController::goToBeamlineStatusView(AMControl *control)
 {
-	if (beamStatusView_) {
+	if (beamlineStatusView_) {
```

Comment:
It's a pedantic detail. But the 'beam status' is different than the 'beamline status', which is something we may implement later. Is it possible to keep the original names here?